packages/sdk-react/src/components/payables/PayablesTable/hooks/usePaymentHandler.tsx

```diff
@@ -51,20 +41,61 @@ export const usePaymentHandler = (
     }
   );
 
-  const markInvoiceAsPaid = async (payableId: string) => {
-    if (payableId) {
-      await payMutation.mutateAsync(
-        {
-          path: { payable_id: payableId },
-        },
-        {
-          onSuccess: (payable) => {
-            toast.success(
-              t(i18n)`Payable "${payable.document_id}" has been paid`
-            );
+  const {
+    executeWithRetry: executeWithRetryMarkPaid,
+    isRetrying: isRetryingMarkPaid,
+  } = useRetry({
+    maxAttempts: 5,
+    initialDelayMs: 1000,
+    backoffFactor: 1.5,
+    shouldRetry: () => {
+      if (!payableId) return true;
+      return true;
```

Comment:
Always true, is this intentional?

Reply:
This was a partial implementation where I intended to add different behavior based on payableId but this should be aligned based on manual creation flow